cluster/examples/kubernetes/ceph/object-user.yaml

```diff
@@ -11,3 +11,12 @@ metadata:
 spec:
   store: my-store
   displayName: "my display name"
+  #maxBuckets: 100
+  #maxSize: "10G"
+  #maxObjects: 10000
+  #caps:
+  #  user: "*"
+  #  bucket: "*"
+  #  metadata: "*"
+  #  usage: "*"
+  #  zone: "*"
```

Comment:
Follow the convention of adding both a hash and a space before comments (`# `), not just a hash. Using Ctrl+/ on VSCode will do this automatically, as will many other editors.